test/integration/etcd/server.go

```diff
@@ -53,6 +54,12 @@ import (
 	_ "k8s.io/kubernetes/pkg/controlplane"
 )
 
+const ecdsaPrivateKey = `-----BEGIN EC PRIVATE KEY-----
```

Comment:
Add a comment to all the places we're inlining private keys that they are for test purposes only and are not considered secure (`This key is for testing purposes only and is not considered secure.`)

We've had issues with people reporting keys in testdata as security vulnerabilities

Reply:
done